frontend/src/text_resources.js

```diff
@@ -347,10 +347,10 @@ let LOCALIZE = new LocalizedStrings({
         orgChartInstructions:
           "Below is a tree view of the organization chart. Once selected, you can use the arrow keys to navigation, expand, and collapse information.",
         backgroundInformation: {
-          title: "Background Information"
+          title: "Overview"
```

Comment:
Are you missing the updates on the French part of this variable?

Reply:
The french translation was already `Contexte` which I thought was right?